app/code/community/HE/TwoFactorAuth/Model/Observer.php

```diff
@@ -260,7 +260,18 @@ protected function _setPasswordData($observer)
         */
         $password = $observer->getEvent()->getPassword();
         $user->setOrigData('password', $password);
-        $user->setPassword();
+        /*
+            Add version compare to handle the different _beforeSave functionality in
+            Mage_Admin_Model_user.
+            In versions below 1.14.1.0, setPassword will assign the admin password to NULL,
+            therefore only allowing the admin user login once.
+            Checking for Enterprise occurs in the callAdminAuthenticate function
+         */
```

Comment:
There's no official PSR on this, but multiline comments should generally be as follows

```
/**
 * This is a multiline comment
 */
```

PSR-5 is still in draft but will define like that https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#3-definitions